From dfd3442b86d1d08fc92da556bf9e970c77985ba9 Mon Sep 17 00:00:00 2001
From: Marek Kasik <mkasik@redhat.com>
Date: Mon, 8 Apr 2013 15:55:52 +0200
Subject: [PATCH] Update man page

Short form of "--page-index" is "-i" not "-p".
Add description of "--named-dest" to man page.
---
 data/evince.1 | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/data/evince.1 b/data/evince.1
index d398b97..9add86a 100644
--- a/data/evince.1
+++ b/data/evince.1
@@ -8,7 +8,8 @@ The Evince Document Viewer application for the GNOME desktop environment
 .B evince
 .RI [--help]
 .RI [--page-label=PAGE\ |
-.RI --page-index=NUMBER]
+.RI --page-index=NUMBER\ |
+.RI --named-dest=DEST]
 .RI [--fullscreen]
 .RI [--presentation]
 .RI [--preview]
@@ -32,9 +33,12 @@ Prints the command line options.
 \fB\-p, \-\-page\-label=PAGE\fR
 Open the document on the page with the specified page label (or page number).
 .TP
-\fB\-p, \-\-page\-index=NUMBER\fR
+\fB\-i, \-\-page\-index=NUMBER\fR
 Open the document on the page with the specified page index (this is the exact page number, not a page label).
 .TP
+\fB\-n, \-\-named\-dest=DEST\fR
+Open the document on the specified named destination.
+.TP
 \fB\-f, \-\-fullscreen\fR
 Run evince in fullscreen mode.
 .TP
-- 
1.8.2

